docs/src/main/sphinx/security.rst

```diff
@@ -2,18 +2,41 @@
 Security
 ********
 
+Cluster access security
+=======================
+
+.. toctree::
+  :maxdepth: 1
+
+  security/tls
+  security/ldap
+  security/password-file
+  security/salesforce
+
+User name management
```

Comment:
User management ?

Reply:
Could go with that. I was just trying to match what SEP docs have now.

Reply:
This is likely fine for now, but as we add more group providers, we will likely need to reorganize